internal/server/agentserver/grpc.go

```diff
@@ -0,0 +1,168 @@
+package agentserver
```

Comment:
Isn't this file a duplicate of internal/api_server/agentserver/grpc.go ?

Reply:
Oh probably, this should have been deleted in rebase, will fix.